contracts/.env.example

```diff
@@ -0,0 +1,10 @@
+# Ethereum config
+PRIVATE_KEY=
+ETHEREUM_RPC_URL=
+ETHERSCAN_API_KEY=
+
+# TendermintX config
+NEXT_HEADER_FUNCTION_ID=
+HEADER_RANGE_FUNCTION_ID=
+GENESIS_HEIGHT=
+GENESIS_HEADER=
```

Comment:
please add in the readme how to get this information to set it up

Reply:
added